src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,11 +1,33 @@
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../services/user';
+import { User } from '../../types/User';
+
+type Props = {
+  todo: Todo;
+  setTodo: (todo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, setTodo }) => {
+  const [user, setUser] = useState<User>();
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getUser(todo.userId)
+      .then(data => {
+        setUser(data);
+      })
+      .catch()
```

Comment:
The `catch` block for the `getUser` promise is empty. Consider adding error handling logic here to manage potential errors when fetching user data.